src/AoCHelper/AoCHelper.csproj

```diff
@@ -25,8 +25,8 @@
   </ItemGroup>
 
   <ItemGroup Label="https://github.com/dotnet/roslyn/issues/45510">
-    <Compile Remove="IsExternalInit.cs" Condition="'$(TargetFramework)' == 'net8.0' OR '$(TargetFramework)' == 'net9.0'" />
-    <Compile Remove="Range.cs" Condition="'$(TargetFramework)' == 'net8.0' OR '$(TargetFramework)' == 'net9.0' OR '$(TargetFramework)' == 'netstandard2.1'" />
+    <Compile Remove="IsExternalInit.cs" Condition="'$(TargetFramework)' == 'net8.0' OR '$(TargetFramework)' == 'net9.0' OR '$(TargetFramework)' == 'net10.0'" />
```

Comment:
This was the bit missing in the update, which caused the warnings you saw locally @codemonkey85.